chaos_kitten/brain/orchestrator.py

```
@@ -177,6 +179,51 @@ async def plan_attacks(state: AgentState, app_config: Dict[str, Any]) -> Dict[st
+        # Modify logic
+        console.print("[cyan]Enter new payload (JSON format):[/cyan]")
+        new_payload_str = Prompt.ask("", default=json.dumps(payload) if payload else "{}")
+        try:
+            new_payload = json.loads(new_payload_str)
+            attack_copy = attack.copy()
+            if attack_copy.get("body"):
+                attack_copy["body"] = new_payload
+            else:
+                 attack_copy["payload"] = new_payload
+            console.print("[green]Payload updated![/green]")
+            return attack_copy
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)